libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -206,15 +300,36 @@ class AP_Airspeed
         Airspeed_Calibration calibration;
         float last_saved_ratio;
         uint8_t counter;
+
+        // Blending error (weight of sensor in blending)
+        float blending_factor;
     } state[AIRSPEED_MAX_SENSORS];
 
+    // Blended airspeed
+    AP_Int8 blend_conf;
+
+    float raw_airspeed_blend;
+    float airspeed_blend;
+    float last_pressure_blend;
+    float filtered_pressure_blend;
+    float corrected_pressure_blend;
+    float EAS2TAS_blend;
+    float temperature_blend;
+    float ratio_blend;
+    float offset_blend;
+
     // current primary sensor
     uint8_t primary;
     
     void read(uint8_t i);
     float get_pressure(uint8_t i);
     void update_calibration(uint8_t i, float raw_pressure);
     void update_calibration(uint8_t i, const Vector3f &vground, int16_t max_airspeed_allowed_during_cal);
+    
+    void process_no_blend(void);
 
     AP_Airspeed_Backend *sensor[AIRSPEED_MAX_SENSORS];
+
+    AP_AHRS *_ahrs;
+    bool _use_blended;
```

Comment:
I got confused while reading this PR thinking "_use_blended" was the param. Can we rename to "blend_active".

Reply:
Agree